CHANGELOG.md

```diff
@@ -35,32 +36,6 @@
 -   Adds component for consignments todo  - orta
 -   Adds storybooks for consignments  - orta
 -   Adds some form elements for consignments - orta
-#### Emission
-
-- [Messaging] Implements pagination for messages - sarah
-- [Messaging] Adds an ArtworkPreview to each Conversation - sarah
-- [Messaging] Adds inbox zero state - maxim
-- [Messaging] Relay support in Conversation container - sarah
-- [Messaging] Adds a FlatList for Messages in Conversations - sarah
-- [dev] Applies prettier, and adds it to the webhooks + extension settings - orta
-- [dev] Adds stories for new messaging components - maxim
-- [Messaging] Adds Conversation, Composer, and Message components - sarah
-- [dev] Upgrades Sentry to 3.0, note Eigen integrator, this may require changes to Eigen - orta
-- [Messaging] Adds Inbox view with real data - luc
-- [dev] Upgrades React Native to v0.45 - sarah
-- [dev] Allows toggling the back button by pressing space - orta
-- Users/Devs can run any PR from inside beta/sim - orta
-- Users now can access stories from inside the app - orta
-
-#### Consignments
-
-- [dev] Adds component for attaching buttons to the keyboard  - orta
-- [dev] Adds component for artist search  - orta
-- [dev] Adds component for consignments todo  - orta
-- [dev] Adds a root component for the Consignments flow  - orta
-- [dev] Adds storybooks for consignments  - orta
-- [dev] Adds some form elements for consignments - orta
-- Added the Welcome/Overview screen for consignments - orta
```

Comment:
twice the entries in the changelog, twice the visible amount of work, twice the $$ ??